### PR TITLE
Add esm.sh to Deno Friendly CDNs

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: advanced/typescript/types.md [hunk 1]
```diff
@@ -241,10 +241,12 @@ declarations just like they provide bundles of JavaScript as ESM.
 
 There are CDNs which host JavaScript modules that integrate well with Deno.
 
-- [Skypack.dev](https://docs.skypack.dev/skypack-cdn/code/deno) is a CDN which
-  provides type declarations (via the `X-TypeScript-Types` header) when you
-  append `?dts` as a query string to your remote module import statements. For
-  example:
+- [esm.sh](https://esm.sh) is a CDN which provides type declarations (via the
```
Comment (ije) [inline advanced/typescript/types.md:244]: esm.sh is a CDN which provides type declarations by default (via the `X-TypeScript-Types` header). You can disable it by adding `?no-dts` query:
```ts
  import React from "https://esm.sh/react?no-dts";
```

**Review by ije — APPROVED** [review]
LGTM!

**Comment by ije** [comment]
@kt3k can you approve this? thanks 🙏

**Review by kt3k — APPROVED** [review]
LGTM 👍

**Pull request merged**